drivers/volume/kdmp/kdmp.go

```diff
@@ -205,22 +205,25 @@ func getZones(pv *v1.PersistentVolume) ([]string, error) {
 	}
 
 	if awsCluster {
-		ebsName := storkvolume.GetEBSVolumeID(pv)
```

Comment:
The way we are determining if this is a AWS or a GCE cluster in getCloudProvider is incorrect. Right now we are making hard checks if it is an EKS cluster or a GKE cluster, but you can also deploy an OCP cluster in AWS. Instead we should be looking at the topology labels by EBS
```
topology.ebs.csi.aws.com/zone=us-east-1d
```

There should be something similar for GCE as well.

Reply:
Made changes as follows
1. For AWS cluster will check `alpha.eksctl.io/cluster-name` or `topology.ebs.csi.aws.com/zone` is set.  For native EKS, topology is  `topology.kubernetes.io/region`, not sure if any other providers would have the same. Checking for `alpha.eksctl.io/cluster-name` seems better check.

2. For GKE will check `topology.gke.io/zone`.

But one open item is if we have OCP on GKE, not sure what the labels would be.  For now, pushing the changes, once we have OCP deployed on GKE can add that label as separate PR.